README.md

```diff
@@ -30,259 +30,25 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Hello World Microverse</b></h3>
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
 
-# 📗 Table of Contents
-
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# Hello World Project
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+** Hello Microverse ** is a first project that project for new students that helps them to understand the concept of version control, GitHub flow and open pull request.  
 
 ## 🛠 Built With <a name="built-with"></a>
```

Comment:
- [ ]  I kindly suggest that you name your branch name after a feature or action e.g `created-Hello-Microverse`. Please kindly check [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) for how to rename your branch or you can create a new feature branch and pull request and use the name of a feature for your branch e.g created-Hello-Microverse for your branch. Kindly ensure to use a very appropriate branch name, `feature` is to vague 🙂. When you create a new branch, ensure you have a link to this pull request in the new pull request to enable the new reviewer follow through with the previous review.
<img width="74" alt="image" src="https://user-images.githubusercontent.com/41163571/234655882-fb76bfaf-b908-450e-b6ba-b69708377eb9.png">